src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -79,12 +79,12 @@
 			<!--#endregion-->
 
 			<!--#region Typo-->
-			<StaticResource x:Key="OutlinedTextBoxFontFamily" ResourceKey="MaterialMediumFontFamily" />
+			<StaticResource x:Key="OutlinedTextBoxFontFamily" ResourceKey="BodyLargeFontFamily" />
 			<StaticResource x:Key="OutlinedTextBoxFontWeight" ResourceKey="BodyLargeFontWeight" />
 			<StaticResource x:Key="OutlinedTextBoxFontSize" ResourceKey="BodyLargeFontSize" />
 			<StaticResource x:Key="OutlinedTextBoxCharacterSpacing" ResourceKey="BodyLargeCharacterSpacing" />
 
-			<StaticResource x:Key="FilledTextBoxFontFamily" ResourceKey="MaterialMediumFontFamily" />
+			<StaticResource x:Key="FilledTextBoxFontFamily" ResourceKey="BodyLargeFontFamily" />
```

Comment:
@kazo0 Documentation should be updated as well I think ;) 
If you can double-check all the controls, please
Example with TextBox here:
![image](https://github.com/unoplatform/Uno.Themes/assets/16295702/d2846ef7-792a-488d-92f1-a40e5180338b)